test/load-generator/config/integration-test-config.json

```diff
@@ -13,6 +13,7 @@
     "directoryURL": "http://boulder:4001/directory",
     "domainBase": "com",
     "challengeStrategy": "random",
+    "postAsGet": false,
```

Comment:
I don't have a strong preference. If I make it default there's basically no reason to keep any of the logic for making GET requests or tracking their rate.

Reply:
I removed this option in 4791b45 and made using POST-as-GET unconditional. I left all of the GET stuff in-place. It can be cleaned up in a separate PR.